packages/client/src/views/SysAdmin/Config/Integrations/Integrations.tsx

```diff
@@ -0,0 +1,97 @@
+import { messages } from '@client/i18n/messages/views/formConfig'
+import { SecondaryButton } from '@client/../../components/lib/buttons'
+import {
+  ListViewItemSimplified,
+  ListViewSimplified
+} from '@opencrvs/components/lib/ListViewSimplified'
+import { Frame } from '@opencrvs/components/lib/Frame'
+import { Navigation } from '@client/components/interface/Navigation'
+import { Header } from '@client/components/Header/Header'
+import { Plus, VerticalThreeDots } from '@client/../../components/lib/icons'
+import { Pill, ToggleMenu } from '@client/../../components/lib'
+import { constantsMessages } from '@client/i18n/messages'
+
+interface IIntegration {
+  name: string
+  status: string
+  systemId: string
+}
+
+export const statuses = {
+  PENDING: 'pending',
+  ACTIVE: 'active',
+  DISABLED: 'disabled',
+  DEACTIVATED: 'deactivated'
+}
+
+export function IntegrationList() {
+  const intl = useIntl()
+
+  return (
+    <Frame
+      header={<Header />}
+      navigation={<Navigation loadWorkqueueStatuses={false} />}
+      skipToContentText={intl.formatMessage(
+        constantsMessages.skipToMainContent
+      )}
+    >
+      <Content
+        title={intl.formatMessage(messages.integrations)}
+        topActionButtons={[
+          <SecondaryButton
+            key="1"
+            id="myButton"
+            onClick={function noRefCheck() {}}
+          >
+            <Plus /> Create Client
+          </SecondaryButton>
+        ]}
+      >
+        For each new client that needs to integrate with OpenCRVS you can create
+        unique client IDs. A number of integration use cases are currently
+        supported, based on both API and webhook technologies.
+        <ListViewSimplified>
+          <ListViewItemSimplified
+            actions={
+              <>
+                <Pill label="Active" type="active" />
+                <ToggleMenu
+                  id="toggleMenu"
+                  menuItems={[
+                    { handler: () => {}, label: 'Reveal Keys' },
+                    { handler: () => {}, label: 'Disable' },
+                    { handler: () => {}, label: 'Delete' }
+                  ]}
+                  toggleButton={<VerticalThreeDots />}
+                />
+              </>
+            }
+            label="Sweet Health"
+            value="Health Integration"
+          />
+          <ListViewItemSimplified
+            actions={
+              <>
+                <Pill label="Active" type="active" />
+                <ToggleMenu
+                  id="toggleMenu"
+                  menuItems={[
+                    { handler: () => {}, label: 'Reveal Keys' },
+                    { handler: () => {}, label: 'Disable' },
+                    { handler: () => {}, label: 'Delete' }
+                  ]}
+                  toggleButton={<VerticalThreeDots />}
+                />
+              </>
+            }
+            label="Sweet Health"
+            value="Health Integration"
+          />
```

Comment:
could we map these from `window.config.INTEGRATIONS` ?

similar to this;
```
window.config.INTEGRATIONS.map(() => (
          <ListViewItemSimplified
            actions={
              <>
                <Pill label="Active" type="active" />
                <ToggleMenu
                  id="toggleMenu"
                  menuItems={[
                    { handler: () => {}, label: 'Reveal Keys' },
                    { handler: () => {}, label: 'Disable' },
                    { handler: () => {}, label: 'Delete' }
                  ]}
                  toggleButton={<VerticalThreeDots />}
                />
              </>
            }
            label="Sweet Health"
            value="Health Integration"
          />
 )
```
but the label would come from window.config.INTEGRATIONS `name` and the `<Pill>` would depend on the `status` ? The other place holder values could be left as is :)